js/ui/grid_core/ui.grid_core.utils.js

```diff
@@ -240,7 +240,10 @@ module.exports = (function() {
                     shading: false,
                     message: loadPanelOptions.text,
                     position: {
-                        of: $element
+                        of: function() {
```

Comment:
Guys, I think, set [collision] https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/positionConfig/collision/) option to 'fit' is a more simple way.